auslib/web/admin/swagger/api.yaml

```diff
@@ -1893,90 +1906,970 @@ paths:
+          schema:
+            type: object
+            required:
+              - scheduled_changes
+            allOf:
+              - $ref: "#/definitions/CountModel"
+              - type: object
+                properties:
+                  scheduled_changes:
+                    description: array where each element is a scheduled change for a Release.
+                    type: array
+                    minItems: 0
+                    items:
+                      type: object
+                      allOf:
+                        - $ref: '#/definitions/SCChangeType'
+                        - $ref: '#/definitions/SCTime'
+                        - $ref: '#/definitions/SCIdModel'
+                        - $ref: '#/definitions/SCDataVersion'
+                        - $ref: '#/definitions/SCGetModel'
+                        - $ref: '#/definitions/DataVersionNullable'
+                        - $ref: '#/definitions/ProductNullable'
+                        - $ref: '#/definitions/ReleaseReadOnly'
+                        - $ref: '#/definitions/ReleaseName'
+                        - type: object
+                          properties:
+                            data:
+                              description: json blob containing information about the release
+                              type: ["object", "null"]
+                              example:
+                                hashFunction: sha512
+                                name: CDM-15
+                                product: Firefox
+                                schema_version: 1000
+                                vendors:
+                                  gmp-eme-adobe:
+                                    platforms:
+                                      WINNT_x86_64-msvc-x64:
+                                        alias: WINNT_x86_64-msvc
+                                    version: '15'
+                      required:
+                        - change_type
+                        - data_version
+                        - when
+                        - product
+                        - name
+                        - data
+                        - read_only
+    post:
+      summary: create scheduled change for a release.
+      description: >
+        Create scheduled change for a release(existing or new). [Docs](http://mozilla-balrog.readthedocs.io/en/latest/admin_api.html#id34)
+      tags:
+        - Scheduled Changes
+      operationId: auslib.web.admin.views.mapper.scheduled_changes_releases_post
+      consumes:
+        - application/json
+      produces:
+        - text/html
+        - application/json
+      externalDocs:
+        url: "http://mozilla-balrog.readthedocs.io/en/latest/admin_api.html#scheduled-changes-object"
+        description: "create scheduled change for releases."
+      parameters:
+        - name: sc_permission_body
```

Comment:
what should I change it into ? BTW in case of body parameters , 'name' is entirely useless and is only needed for uniquely identifying parameter name by swagger but never used in handling the request.json in python code.